src/Oauth1.php

```diff
@@ -62,29 +59,40 @@ class Oauth1 implements SubscriberInterface
      */
     public function __construct($config)
     {
-        $this->config = Collection::fromConfig($config, [
+        $this->config = [
```

Comment:
Could this just be `$this->config = $config + [ ...`? This does a union of the two arrays.